backslash/contrib/slash_plugin.py

```diff
@@ -145,9 +144,37 @@ def session_interrupt(self):
         if self.session is not None:
             self.session.report_interrupted()
 
+    @slash.plugins.registers_on(None)
+    @handle_exceptions
+    def report_tests(self, tests):
```

Comment:
I think this should be called `report_planned_tests` to avoid confusion (or `report_upcoming_tests` to match the rest of the changes)